cmd/traffic/cmd/manager/service.go

```diff
@@ -459,9 +466,16 @@ func (m *Manager) WatchIntercepts(session *rpc.SessionInfo, stream rpc.Manager_W
 func (m *Manager) PrepareIntercept(ctx context.Context, request *rpc.CreateInterceptRequest) (*rpc.PreparedIntercept, error) {
 	ctx = managerutil.WithSessionInfo(ctx, request.Session)
 	dlog.Debugf(ctx, "PrepareIntercept called")
+
 	span := trace.SpanFromContext(ctx)
 	tracing.RecordInterceptSpec(span, request.InterceptSpec)
-	return m.state.PrepareIntercept(ctx, request)
+
+	pi, err := m.state.PrepareIntercept(ctx, request)
+	if err != nil {
+		return nil, err
+	}
+
+	return pi, nil
```

Comment:
I don't understand this change. It doesn't add anything.

Reply:
ah yeah this is left over from when i was doing mode checks at the very end and needed to check this error. will remove